### PR TITLE
Cgo#13 building selection

### LINKED ISSUE #13: Concordia Building Selection
# User Story 

## WHO/WHAT/WHY
* As a user, I want to be able to see all the SGW and Loyola buildings in a menu so that I can easily click on them and go to the building 

## ACCEPTANCE CRITERIA
* Menu to see all SGW and Loyola Buildings
* Clicking on any of them will bring you to the building

## IMPLEMENTATION NOTES
* SGW and Loyola will show different lists

### PR DESCRIPTION
**Pull request by danielwiktorczyk**
Ready for QA review! Scoped out the search bar since it is low priority, and can be introduced later as an enhancement. Linked issue: #13

### CODE REVIEW

File: android/app/build.gradle
Comment (DBatistaFerreira) [inline android/app/build.gradle]: Why?

**Review by DBatistaFerreira — COMMENTED** [review]
Why do we need a loyola_campus_building_menu and a sgw_campus_building_menu? Can't we just have 1 class and a way to know which campus it is and then load the proper buildings? Or is it because they're 2 tabs that it needs to be that way?

File: test/smoke_test.dart
Comment (DBatistaFerreira) [inline test/smoke_test.dart:1]: Nicely done with this :)

File: android/app/build.gradle
Comment (danielwiktorczyk) [reply android/app/build.gradle]: I was having issues with this, my environment couldn't find the "GradleException" class. I can remove this from the PR if it works on your end fine

File: test/smoke_test.dart
Comment (danielwiktorczyk) [reply test/smoke_test.dart:1]: Thanks!

**Comment by danielwiktorczyk** [comment]
@Pwnagebird , for the campus tab classes, I made them separately since I'm not sure if that's possible, and because this campus building list shouldn't scale larger than the loyola and sgw elements. When we look at technical debt later, time permitting, this can be looked at then. Good comment though!

**Comment by DBatistaFerreira** [comment]
@danielwiktorczyk Alright, due to time constraints lets just move on since it works. We will refactor it later on :)

**Review by DBatistaFerreira — APPROVED** [review]

**Review by MoreWaqar — CHANGES_REQUESTED** [review]
All in all limited cross over with other files so low risk of damage to other existing processes. Functionality is testable, but missing data entry that will fix remaining issues.

**Review by MoreWaqar — APPROVED** [review]
Request was partially met. No problems encountered and the new code is well structured.

Recommend to wait for review by Nick, but this looks good otherwise.

**Comment by neliop** [comment]
The menus look really good, Daniel! Nice job! Just a design suggestion, but maybe the title of the page can be in the app bar (instead of having the logo on every screen). Anyway, feel free to merge.

**Review by neliop — APPROVED** [review]

**Comment by danielwiktorczyk** [comment]
Ready to be merged!

Notes:
- Confirmed with @MoreWaqar that all desired data was indeed added, and that request was actually fully met. 
- Minor refactoring can be done at a later date

**Pull request merged**
